subsys/bluetooth/mesh/shell/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_BT_MESH_SHELL shell_model_utils.c)
```

```suggestion
zephyr_library_sources(shell_model_utils.c)
```
since this entire folder is protected by that config